### PR TITLE
[python-api] - add preselect params to Dialog.multiselect() and Dialog().select()

### PR DESCRIPTION
**Pull request by phil65**
Requested here for example: http://forum.kodi.tv/showthread.php?tid=270814
I also have some usecases for this.

### CODE REVIEW

File: xbmc/interfaces/legacy/Dialog.h [hunk 1]
```diff
@@ -96,29 +96,31 @@ namespace XBMCAddon
...
        * example:\n
        *   - dialog = xbmcgui.Dialog()\n
        *   - ret = dialog.select('Choose a playlist', ['Playlist #1', 'Playlist #2, 'Playlist #3'])\n\n
        */
-      int select(const String& heading, const std::vector<String>& list, int autoclose=0);
+      int select(const String& heading, const std::vector<String>& list, int autoclose=0, int preselect=-1);
 
       /**
        * multiselect(heading, list [,autoclose]) -- Show a multi-select dialog.
        *
```
Comment (MartijnKaijser) [inline xbmc/interfaces/legacy/Dialog.h]: This needs updating as well. Perhaps also add example how that preselect list should look like

**Comment by phil65** [comment]
jenkins build this please.
@tamland, ok to merge?

**Comment by ronie** [comment]
@phil65 kodi is crashing on my end if i don't provide a preselect param in Dialog().multiselect().
can you confirm?

http://xbmclogs.com/pol4onkzz

**Comment by phil65** [comment]
Hmm didnt notice any crashing while testing. will take a look.

**Pull request merged**